source/geometry_model/box.cc

```diff
@@ -228,6 +228,19 @@ namespace aspect
       return extents;
     }
 
+    template <int dim>
+    std::array<unsigned int,dim>
+    Box<dim>::get_repetitions () const
+    {
+      std::array<unsigned int,dim> rep;
+      for (unsigned int i=0; i<dim; ++i)
+        {
+          rep[i] = repetitions[i];
+        }
+
+      return rep;
```

Comment:
If you change the declaration of this function as suggested above, you can do this instead:
```suggestion
      return repetitions;
```

Reply:
Ah, I did not see that `repetitions` is of type `unsigned int [3]`, which is awkward to use. Keep your version for now, I will make a separate PR to change it later.

Reply:
If you rebase, you'll get @gassmoeller's changes made in #5098 that make it possible what he suggested.